frontend/interfaces/software.ts

```diff
@@ -5,4 +5,11 @@ export default PropTypes.shape({
   name: PropTypes.string,
   version: PropTypes.string,
   id: PropTypes.number,
+  vulnerabilities: PropTypes.shape({
+    id: PropTypes.number,
+    uid: PropTypes.number,
+    username: PropTypes.string,
+    type: PropTypes.string,
+    groupname: PropTypes.string,
+  }),
```

Comment:
I think it should be an array of objects.
```suggestion
  vulnerabilities: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.number,
    uid: PropTypes.number,
    username: PropTypes.string,
    type: PropTypes.string,
    groupname: PropTypes.string,
  })),
```